src/actors_extended.c

```diff
@@ -54,18 +54,18 @@ void func_802B02B4(struct ShellActor *shell, s32 shellType) {
 
     switch(shellType) {
         case ACTOR_GREEN_SHELL:
-            shell->state = 5;
+            shell->state = GREEN_SHELL_HIT_A_RACER;
             break;
         case ACTOR_RED_SHELL:
-            shell->state = 7;
+            shell->state = DESTROYED_SHELL;
             break;
     }
 }
 
 // Sets velocities for a banana, used when a racer runs into
 // a banana bunch.
 void func_802B039C(struct BananaActor *banana) {
-    banana->state = 1;
+    banana->state = DROPPED_BANANA;
```

Comment:
Wait. We were talking about placed banana earlier weren't we?

What's going on between dropped, placed, and ground?

I guess ground is like "Hey, this banana is actually touching the ground"

Hmm... I wonder if there's a better way to write that comment. Like // Banana is touching the track or collided.

If such a specific distinguishment exists, please specify in one of the comments so as to not confuse the two.